apps/server/cypress/e2e/suite2/unsupportedProvider.cy.ts

```diff
@@ -0,0 +1,9 @@
+describe('unsupported provider', () => {
```

Comment:
Something unexplainable was happening but after trying a bunch of different things it seems like it's working now. It looked like it would find some elements before fetch requests would finish but then a fetch would finish and put the dom back into a loading state, so it wouldn't find MX Bank because it was loading. But now when I test it locally, it seems to work.
![image](https://github.com/Universal-Connect-Project/ucw-app/assets/8431132/a4ed5727-e94f-4149-bc04-16149d2758d8)